packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -651,13 +651,17 @@ function getCustomInputsResponses(
   return customInputsResponses;
 }
 
-async function getTeamId({ eventType }: { eventType: Awaited<ReturnType<typeof getEventTypesFromDB>> }) {
+export async function getTeamId({
+  eventType,
+}: {
+  eventType: { team: { id: number | null } | null; parentId: number | null } | null;
```

Comment:
nit: what's the need for allow **null** eventType

Also, even if we need to do that. A cleaner approach would be to just check eventType for nullishness and return early, so that later code need not use conditional chain on it.

Reply:
yeah that's a good point will make the change